doc/languages-frameworks/python.section.md

```diff
@@ -392,11 +395,11 @@ and adds it along with a `numpy` package to a Python environment.
 with import <nixpkgs> {};
 
 ( let
-    my_toolz = python38.pkgs.buildPythonPackage rec {
+    my_toolz = python39.pkgs.buildPythonPackage rec {
       pname = "toolz";
       version = "0.10.0";
 
-      src = python38.pkgs.fetchPypi {
+      src = python39.pkgs.fetchPypi {
         inherit pname version;
```

Comment:
Their is a leftover in this expression `python38.withPackages (ps: [ps.numpy my_toolz])` Is it intentional ? Now It can be udpated to 3.10

Reply:
No, wasn't intentional. I apparently missed it.